### PR TITLE
add getClaimDayNums that gets data in parallel

### PR DESCRIPTION
**Pull request by 0xBrian**

### CODE REVIEW

**Comment by 0xBrian** [comment]
@yellowgh0st OK, this seems to be very fast.

**Comment by yellowgh0st** [comment]
ok i'm gonna check that now

File: src/common/ethereum.js [hunk 1]
```diff
@@ -152,6 +152,7 @@ const claimShare = async (era, day, provider) => {
 	return await contract.withdrawShare(era, day)
 }
 
+
```
Comment (yellowgh0st) [inline src/common/ethereum.js]: Extra newline

**Review by yellowgh0st — CHANGES_REQUESTED** [review]
Looks good!

File: src/common/ethereum.js (hunk 1 above)
Comment (0xBrian) [reply src/common/ethereum.js]: Got it.

**Review by yellowgh0st — APPROVED** [review]

**Pull request merged**